types/sqlite3/index.d.ts

```diff
@@ -89,6 +89,7 @@ export class Database extends events.EventEmitter {
     on(event: string, listener: (...args: any[]) => void): this;
 
     configure(option: "busyTimeout", value: number): void;
+    interrupt(): void;
```

Comment:
It may throw an error.
For example, error has happened when database is not opened:

```console
$ node index.js
/Users/user/work/js/test-sqlite3/index.js:3
db.interrupt();
   ^

Error: Database is not open
    at Object.<anonymous> (/Users/user/work/js/test-sqlite3/index.js:3:4)
    at Module._compile (internal/modules/cjs/loader.js:1121:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1160:10)
    at Module.load (internal/modules/cjs/loader.js:976:32)
    at Function.Module._load (internal/modules/cjs/loader.js:884:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:67:12)
    at internal/main/run_main_module.js:17:47
```

Source: 

```js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database(':memory:');
db.interrupt();
```

Reply:
It does not cause an error.

```js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database(':memory:');
db.each('select 1', err => {
  db.interrupt();
});
```